udpSource/src/main/java/io/vantiq/extsrc/udp/ConfigurableUDPSource.java

```diff
@@ -497,11 +496,16 @@ static boolean isValidServer(Object server) {
      * @param config    The {@link Map} obtained from the config file
      */
     static void setupServer(Map config) {
-        targetVantiqServer = config.get("targetServer") instanceof String ? (String) config.get("targetServer") :
-                "wss://dev.vantiq.com/api/v1/wsock/websocket";
         MAX_UDP_DATA = config.get("maxPacketSize") instanceof Integer ? (int) config.get("maxPacketSize") : 1024;
         LISTENING_PORT = config.get("defaultBindPort") instanceof Integer ? (int) config.get("defaultBindPort") :
                 3141;
+        
+        if (config.get("targetServer") instanceof String) {
+            targetVantiqServer = (String) config.get("targetServer") ;
+        } else {
+            throw new RuntimeException("Missing authentication token in config file. Please place in 'authToken'.");
```

Comment:
I think this is a cut & paste error -- should be asking for VANTIQ server not auth token, methinks.